lib/MusicBrainz/Server/Data/Search.pm

```diff
@@ -942,7 +948,15 @@ sub external_search
         $pager->entries_per_page($limit);
         $pager->total_entries($total_hits);
 
-        return { pager => $pager, offset => $offset, results => \@results, last_updated => $last_updated };
+        return {
+            pager => $pager,
+            offset => $offset,
+            results => \@results,
+            last_updated => $last_updated,
+            $show_largest_available_page ? (
+                largest_available_page => $largest_available_page
+            ) : ()
```

Comment:
Is it possible to determine all of this from the pager given to the component already? e.g. there's `pager.last_page`, does that match `$largest_available_page`?  And does `pager.current_page` match `$page`?

Reply:
`pager.last_page` does give the same as I needed, yes. `pager.current_page` doesn't show the passed parameter though, it is the same as `pager.last_page` if I pass any page number above that number. I could work with that just fine, the only problem is that `pager.last_page` is 1 even if there's no results at all (including in page 1) so if I ask for page 2 of a no-results search, I'd still think "the last page of results is 1".

Reply:
That absolutely does it, so this is a looot simpler now. Thanks!